app/pathogen/arbovirus/dashboard/(banner)/ArboBanner.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
```

Comment:
another nit: We do not need a directory called (banner) since no other files will go in here. You can lift the file up to the dashboard directory and it is essentially the same

Reply:
Done!